how-to/releaseduty/merge-duty/merge_new_esr_branch.rst

```diff
@@ -91,48 +119,57 @@ Task list and known dependencies
     asking for the esrXX repository to be added to Phabricator, as
     mozilla-esrXX, tagged for uplifts, and hooked up to code-review-bot
 
-18. Run a beta as esr simulation to identify permanent build and test failures
+18. Once XX becomes the current beta, run a beta as esr simulation to
+    identify permanent build and test failures. (`./mach try release
+    --disable-pgo -v XX.0esr --migration beta-to-release --migration
+    release-to-esr --tasks release-sim`). Don't hesitate to ask sheriffs for
+    help with classification of tests failures.
+
+19. Early in the XX nightly cycle, run a ESRXX staging release to identify
```

Comment:
We might want to reorder these?  Effectively these can run just after adding support in gecko_taskgraph.  (It's also a bit odd to have a "once XX is beta" step before an "early in XX nightly" one.)

Reply:
Well you probably don't want the release sim ran while in nightly in case new tests start failing. But yeah, there's nothing stopping you from doing so as long as the gecko taskgraph update is in.

I didn't reorder steps for 2 reasons:

- The order is already wrong. As long as there's no dependency marked in the step you can probably do it. (22. for example is the shipit update which I did at the same time as 3.)
- I don't want to mess up dependencies. Moving a step early means updating a lot of numbers and I'm sure to miss one...

I'll reorder 18/19 because it does look weird but unless you have strong feelings about it, I'll avoid reordering everything